src/nectarchain/dqm/start_calib.py

```diff
@@ -16,11 +16,27 @@
 from trigger_statistics import TriggerStatistics
 from camera_monitoring import CameraMonitoring
 
+import argparse
+
+# Create an ArgumentParser object
+parser = argparse.ArgumentParser(description='My script')
+parser.add_argument('-plot', action='store_true', help='Whether to plot the data')
+parser.add_argument('input_paths', nargs='+', help='Input paths')
```

Comment:
So then:
```
from nectarchain.calibration.container import utils
[...]
parser.add_argument('runnb', help='Run number')
[...]
dm = utils.DataManagement()
dm.findrun(args.runnb)
```

Reply:
I would put that in the version that will run automatically, but here if I just want to choose specific files from a run as input (for example only files with extension 0000) it won't work no ?

Reply:
Indeed, the oldest runs with previous nomenclature would not run. In order to avoid having different concurrent versions of the same code, I would then suggest to make such a `runnb` argument optional, use `nectarchain.calibration.container.util.DataManagement` in case it is provided, or do as implemented now if not.

Reply:
okay I added the option for now, but it needs to be tested further